### PR TITLE
Upgrade to Rails 5.2

### PR DESCRIPTION
**Pull request by vsmart**
#### Is this solving an issue already in our backlog? If so, please mention which one

#646 

#### Special things about this PR to be considered

#672

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ gem "pry"
 gem "rubocop"
 
 group :development do
-  gem 'sqlite3'
+  gem 'sqlite3', '1.3.13'
```
Comment (vsmart) [inline Gemfile]: This should fix the issue i was seeing - "Gem::LoadError: Specified 'sqlite3' for database adapter, but the gem is not loaded" see  https://github.com/rails/rails/issues/35161 for reference 
cc @lislis

Comment (lislis) [reply Gemfile]: good point, but can you change it to 
```gem 'sqlite3', '~> 1.3.13'``` ?

Comment (vsmart) [reply Gemfile]: done in https://github.com/rubycorns/rorganize.it/pull/673/commits/71fda07c89aaec7d6ca598db0fe7c9b97c9a939b

File: config/environments/production.rb [hunk 2]
```diff
@@ -14,9 +14,14 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
+  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
+  # `config/secrets.yml.key`.
+  config.read_encrypted_secrets = true
```
Comment (vsmart) [inline config/environments/production.rb]: change to false, we're not using this yet

File: config/environments/production.rb [hunk 3]
```diff
@@ -61,24 +66,8 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
-  config.action_mailer.default_url_options = {
```
Comment (lislis) [inline config/environments/production.rb]: I think we still need this to send emails in production

File: config/puma.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Puma can serve each request in a thread from an internal thread pool.
```
Comment (lislis) [inline config/puma.rb]: We use Thin instead of Puma, we can delete this file

**Comment by lislis** [comment]
clap

**Pull request merged**